osquery/main/harnesses/fuzz_sqlquery.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <osquery/config/config.h>
+#include <osquery/core.h>
+#include <osquery/database.h>
+#include <osquery/logger.h>
+#include <osquery/registry.h>
+#include <osquery/sql.h>
+#include <osquery/sql/dynamic_table_row.h>
+#include <osquery/sql/tests/sql_test_utils.h>
```

Comment:
But, why do you need this include?

Reply:
Huh; I guess I don't. I swore I got an error when I omitted it in my initial attempt on a different branch; but now when I remove it, it works...